pom.xml

```
@@ -21,7 +21,7 @@
     <groupId>in.vectorpro.dropwizard</groupId>
     <artifactId>dropwizard-swagger</artifactId>
 
-    <version>2.0.28-2</version>
+    <version>2.0.29</version>
```

As dropwizard version of this bundle continue to remain `2.0.28`, following the naming convention - version of the bundle will upgrade as below:

```suggestion
    <version>2.0.28-3</version>
```

---

done